rails/app/views/welcome/index.html.erb

```diff
@@ -27,4 +27,7 @@
       <%= link_to "en", root_path(locale: "en") %>
     </div>
   </div>
+  <div class="logo">
+	<%= image_tag 'act-matawai.png', alt: 'Amazon Conservation Team and Stichting voor Dorpsontwikkeling Matawai' %>
+  </div>
```

Comment:
This chunk may need additional styles, or be hidden, when the screen is tiny. It does some really strange things to the primary welcome bubble.

Reply:
Thanks for catching this. I worked on the .scss file some more.